Sources/SnapshotPreviewsCore/ExpandingViewController.swift

```diff
@@ -78,7 +78,11 @@ public final class ExpandingViewController: UIHostingController<EmergeModifierVi
 
   public override func viewDidLayoutSubviews() {
     super.viewDidLayoutSubviews()
```

Comment:
Isn't `viewDidLayoutSubviews` only called after the layout pass has completed? `viewWillLayoutSubviews` is called before it's completed, but we aren't using that here